src/load.ts

```diff
@@ -0,0 +1,75 @@
+import has from './has';
+import Promise from './Promise';
+
+declare var require: Function;
+declare var define: any;
```

Comment:
And do we need an interface somewhere, or do we need a typings from the loader that we should depend on?